bot/code_review_bot/mercurial.py

```diff
@@ -32,6 +32,10 @@
 # Wait successive exponential delays: 6sec, 36sec, 3.6min, 21.6min
 PUSH_RETRY_EXPONENTIAL_DELAY = 6
 
+# External services to manage hash reference related to Git repositories
+GIT_TO_HG = "https://lando.moz.tools/api/git2hg/firefox/{}"
+# https://github.com/mozilla-firefox/firefox/
+FIREFOX_GITHUB_COMMIT_URL = "https://api.github.com/repositories/835510315/commits/{}"
```

Comment:
1. The rate limit is 60 requests per hour for unauthenticated requests. It is better to support authentication now.
2. Could `https://api.github.com/repos/mozilla-firefox/firefox/commits/{}` be used instead?